app/views/fragments/analytics/krux.scala.html

```diff
@@ -1,9 +1,10 @@
+<!-- Source: /snippet/controltag?confid=Jglpp88U&site=Subscriptions&edit=1 -->
 <script class="kxct" data-id="Jglpp88U" data-timing="async" data-version="1.9" type="text/javascript">
-    window.Krux || ((Krux=function() { Krux.q.push(arguments) }).q=[]);
-    (function(){
-        var k=document.createElement('script');k.type='text/javascript';k.async=true;
-        var m,src=(m=location.href.match(/\bkxsrc=([^&]+)/))&&decodeURIComponent(m[1]);
-        k.src = /^https?:\/\/([a-z0-9_\-\.]+\.)?krxd\.net(:\d{1,5})?\//i.test(src) ? src : src === "disable" ? "" : (location.protocol==="https:"?"https:":"http:")+"//cdn.krxd.net/controltag?confid=Jglpp88U";
-        var s=document.getElementsByTagName('script')[0];s.parentNode.insertBefore(k,s);
-    }());
-</script>
+  window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);
```

Comment:
@tudorraul Bit late on the feedback here as it's merged, but if the new subscriptions is going to be over HTTPS you'll want to make sure you check for mixed-content warnings from Krux as we have had a lot of problems with that on Membership. It's OK once configured, but something to be aware of.